### PR TITLE
feat: remove extra spaces in `url` inside `@import`

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: tests/css_import/url.css [hunk 1]
```diff
@@ -24,3 +24,7 @@ $dir: 'fonts';
 @import 'rounded-corners', 'text-shadow';
 $family: unquote("Droid+Sans");
 @import url("http://fonts.googleapis.com/css?family=#{$family}");
+@import url( "foo.css");
+@import url("foo.css" );
+@import url( "foo.css" );
+@import url(  "foo.css"  );
```
Comment (j-f1) [inline tests/css_import/url.css]: What about `@import url("foo bar baz.css");`?

File: src/language-css/printer-postcss.js [hunk 2]
```diff
@@ -218,7 +218,7 @@ function genericPrint(path, options, print) {
       return adjustStrings(n.value, options);
     }
     case "media-url": {
-      return adjustStrings(n.value, options);
+      return adjustStrings(n.value.replace(/ /g, ""), options);
```
Comment (lydell) [inline src/language-css/printer-postcss.js]: What about other types of whitespace?

**Comment by evilebottnawi** [comment]
@j-f1 @lydell fixed and add tests :+1:

**Comment by j-f1** [comment]
```
✘  prettier/prettier  Replace `.replace(/^url\(\s+/gi,·"url(")` with `⏎······.replace(/^url\(\s+/gi,·"url(")⏎······`  
  src/language-css/clean.js:97:32
```

**Comment by evilebottnawi** [comment]
@j-f1 fixed
@lydell friendly ping

File: tests/css_import/__snapshots__/jsfmt.spec.js.snap [hunk 3]
```diff
@@ -63,6 +63,16 @@ $dir: 'fonts';
 @import 'rounded-corners', 'text-shadow';
 $family: unquote("Droid+Sans");
 @import url("http://fonts.googleapis.com/css?family=#{$family}");
+@import url( "foo.css");
+@import url("foo.css" );
+@import url( "foo.css" );
+@import url(  "foo.css"  );
+@import url("foo bar baz.css");
+@import url(  "foo bar baz.css"  );
+@import url("foo  bar  baz.css");
```
Comment (lydell) [inline tests/css_import/__snapshots__/jsfmt.spec.js.snap]: There are no tests for the case insensitivity?

Comment (evilebottnawi) [reply tests/css_import/__snapshots__/jsfmt.spec.js.snap]: @lydell right now media parsed don't works with `URL` case, i think better fix this in parser then add tests and fixes

**Pull request merged**